sdv/logging/logger.py

```diff
@@ -10,14 +11,18 @@
 class CSVFormatter(logging.Formatter):
     """Logging formatter to convert to CSV."""
 
-    def __init__(self):
+    def __init__(self, filename=None):
```

Comment:
```suggestion
    def __init__(self, filepath=None):
```
I think this has to be the full path to the file right?

Reply:
Yeah it is the full path, just named `filename` in the handler

Reply:
Ah ok